src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj

```diff
@@ -1,6 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
-<Project>
+﻿<Project>
```

Comment:
Oh, I should also mention that any of the .csproj files I opened, I made the whitespace consistent and some of these files, like this one, have the `xml` header and this copyright comment, which is **not necessary** for non-source files. There are very rare situations that the `xml` header is needed, but none of these .csproj files needed it.